test_scripts/Resumption/Handling_errors_from_HMI/001_successful_resumption_SUCCESS_resultCode.lua

```diff
@@ -0,0 +1,78 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0190-resumption-data-error-handling.md
```

Comment:
@yang1070 Please notice this is an old PR for `0188` proposal that was opened almost 2 years ago. All the tests require significant update since then. So this PR will be closed and new one will one opened.
Please also notice all the deliverables for this proposal are planned to be published for Ford team review at Aug-10.
(For both `0190` and `0188` proposals at the same date).

Reply:
@dboltovskyi what is the new PR for 0188? Has it been created? Or we need to wait till Aug-10 to review the new PR?

Reply:
@yang1070 New PR hasn't created yet since work is still in progress. We appreciate if Ford team waits till Aug-10 when final version is going to be published.

Reply:
@dboltovskyi Thank you